src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -125,21 +125,22 @@ function LibraryServiceClient(gaxGrpc, grpcClients, opts) {
       gax.createByteLengthFunction(grpcClients.google.example.library.v1.Book))
   };
 
-  var operationsClient = new gax.lro({
+  var operationsOpts = {
     servicePath: servicePath,
     port: port,
     sslCreds: sslCreds,
     clientConfig: clientConfig,
     appName: appName,
-    appVersion: appVersion
-  });
+    appVersion: appVersion,
+    scopes: ALL_SCOPES
+  }
+  var operationsClient = new gax.lro(operationsOpts).operationsClient(operationsOpts);
```

Comment:
`gax.lro` will accept a slightly different parameters. There's a little different things (like `keyFilename`) which are passed to `index.js`, not here -- that's a bit complicated.  You'll probably have to modify gax a little bit to allow having those information.

See also https://github.com/googleapis/gax-nodejs/blob/master/lib/grpc.js#L58 grpcClient passes `options` to auth.